src/remoteConnector.ts

```diff
@@ -669,7 +717,7 @@ export class RemoteConnector extends Disposable {
 					await this.showSSHPasswordModal(password, gatewayFlow);
 				}
 
-				this.telemetryService.sendUserFlowStatus('connected', gatewayFlow);
+				this.telemetryService.sendUserFlowStatus('got_destination', gatewayFlow);
```

Comment:
@akosyakov I'd like to change this status to `got_destination` since it's not really connected yet. Good for you?

Reply:
how the funnel is going to look for all modes?

Reply:
> how the funnel is going to look for all modes?

What do you mean? 

For other connect ways (`direct ssh gateway` and `local-app`) they should work like before except naming changed to `got_destination`. 

local-ssh funnel will go:
- Event vscode_desktop_ssh, with property `kind: local-ssh`
- With funnel: `connecting` -> `? local-ssh-failure` -> connected (`got_destination`)

Reply:
let's not do it in this PR

Reply:
> let's not do it in this PR

Revert to `connected` ?